src/test/java/com/cloudbees/jenkins/plugins/advisor/BundleUploadTest.java

```diff
@@ -80,7 +83,9 @@ public void execute() throws Exception {
     // Refresh the configuration?
     assertThat(config.getLastBundleResult(), containsString(BUNDLE_SUCCESSFULLY_UPLOADED));
 
-    assertThat(Files.list(Paths.get(BundleUpload.TEMP_BUNDLE_DIRECTORY)).count(), is(equalTo(0L)));
+    try (Stream<Path> children  = Files.list(Paths.get(BundleUpload.TEMP_BUNDLE_DIRECTORY))) {
```

Comment:
Streams should always be closed to ensure that any resources used by them can be returned.
Mostly people ignore this as a stream of a list of Strings has no locks and its "fine" but when the stream has file system resources this will cause issues.

So close the stream - be nice.